LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 MD. Sakib Khan
```

Comment:
Change the `copyright` issue

Reply:
I selected Licence term while creating repo :P I dont know much about it.
deleted.